.travis.yml

```diff
@@ -7,6 +7,15 @@ os: osx
 # on Traivs.
 env:
   - TEST_DIR=com.google.code.gson-gson
+  - TEST_DIR=org.joda-joda-convert
+  - TEST_DIR=joda-time-joda-time
```

Comment:
I asked britter about this in the other thread. Let's see if he responds with some insights.